kernel/kernel-idgenerator-service/src/main/resources/bootstrap.properties

```diff
@@ -1,7 +1,7 @@
  #spring.cloud.config.uri=localhost
 #spring.cloud.config.label=master
 spring.profiles.active=local
-spring.cloud.config.name=application,kernel
+spring.cloud.config.name=kernel
```

Comment:
why is the application removed ?

Reply:
This change is done by devops team recently in between the code merges and it got removed by my code merge from develop-java21 to develop branch. Reverted it by adding the config name application.